src/AureliaPlugin.ts

```diff
@@ -160,7 +160,9 @@ export class AureliaPlugin {
     if (!dllPlugin && !opts.noWebpackLoader) {
       // Setup aurelia-loader-webpack as the module loader
       // Note that code inside aurelia-loader-webpack performs PLATFORM.Loader = WebpackLoader;
-      this.addEntry(compiler.options, "aurelia-loader-webpack");      
+      // Since this runs very early, before any other Aurelia code, we need "aurelia-polyfills"
+      // for older platforms (e.g. `Map` is undefined in IE 10-).
+      this.addEntry(compiler.options, ["aurelia-polyfills", "aurelia-loader-webpack"]);      
```

Comment:
`aurelia-polyfills` has always been statically imported by `aurelia-bootstrapper`:
https://github.com/aurelia/bootstrapper/blob/master/src/index.js#L1

What this change does is that it ensures it's imported before we run `aurelia-loader-webpack`. No doing so created bugs in browsers without `Map` support.

My plugin enables you to make `aurelia-polyfills` an empty module if you want to use something else, with `feature: { polyfills: "none" }`
https://github.com/aurelia/webpack-plugin/wiki/AureliaPlugin-options#features